theme.js

```diff
@@ -0,0 +1,43 @@
+import { slug } from 'github-slugger'
+import path from 'node:path'
+import {
+  MarkdownTheme,
+  MarkdownThemeRenderContext
+} from 'typedoc-plugin-markdown'
+
+const externalLinkRegex = /^(http|ftp|mailto|tel)s?:\/\//
```

Comment:
```suggestion
const externalLinkRegex = /^\w:/
```

what about using protocol check? `mailto` has no `//` and `mailtos`.

Reply:
allow-list -- you mean just check if link is relative?

Reply:
It feels much simpler, I'll use it 🙂
Thanks!

Reply:
> you mean just check if link is relative?

yes